FileManipulator/FileManipulator.csproj

```
@@ -3,16 +3,16 @@
     <PropertyGroup>
         <RootNamespace>Messerli.FileManipulator</RootNamespace>
         <AssemblyName>Messerli.FileManipulator</AssemblyName>
-    </PropertyGroup>
-    <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
-        <LangVersion>8.0</LangVersion>
+        <TargetFramework>net5.0</TargetFramework>
+        <LangVersion>9.0</LangVersion>
         <Nullable>enable</Nullable>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
     </PropertyGroup>
     <ItemGroup>
-        <ProjectReference Include="..\FileManipulatorAbstractions\FileManipulatorAbstractions.csproj" />
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+            <_Parameter1>$(AssemblyName).Test</_Parameter1>
+        </AssemblyAttribute>
```

This is supported by the .NET 5 SDK:
```suggestion
        <InternalsVisibleTo Include="$(AssemblyName).Test" />
```